.coveragerc

```
@@ -0,0 +1,12 @@
+# This is the configuration for code coverage checks
+# https://coverage.readthedocs.io/en/latest/config.html
+
+[run]
+source = src/example
```

Replace skeleton directory name:
```suggestion
source = src/trustymail
```

---

Ack I thought I'd changed that.